client/blocks/signup-form/index.jsx

```diff
@@ -809,9 +808,13 @@ class SignupForm extends Component {
 	};
 
 	renderWooCommerce() {
+		// As we migrate more inputs to @wordpress/components' <TextControl />,
+		// we should extend this classname and realted styles for better coherency.
+		const inputClassName = 'signup-form__woo-input';
 		return (
 			<div>
```

Comment:
I see you opted to keep CSS changes from the original to a minimum, but I think this would be a lot cleaner if we dropped the input-level class and instead spaced them all with a flex gap on the container div. With just a bit of top padding to compensate.